internal/vds/connection.go

```diff
@@ -6,6 +6,39 @@ import (
 	"net/url"
 )
 
+func srtContainsObject(srt string) bool {
+	return strings.Contains(srt, "o");
+}
+
+func srtContainsContainer(srt string) bool {
+	return strings.Contains(srt, "c");
+}
+
+/** Validate 'srt' (Allowed Resource Types)
+ *
+ * Validate that the sas token has both 'c' (container) and 'o' (object/blob)
+ * in it's srt field.
```

Comment:
Maybe slightly rephrase this to contain information that one needs the `c` and the `o` field for accessing the VDS dataset. 

To me it would make sense to mention here what is checked (str contains `c` and `o`) and what that means (one has access to `container` and object. The reason why we check this could come below where we actually call this function.

Reply:
See if it's better now! If not I'm happy to give it another go 😄

Reply:
Looks good.